<!-- -*- xml -*- -->
<launch>

  <!-- v4l2_camera -->
  <node pkg="v4l2_camera" exec="v4l2_camera_node" name="v4l2_camera_node" output="screen">
  </node>

  <!-- blob_tracker -->
  <node pkg="visp_ros" exec="visp_ros_blob_tracker_node" name="blob_tracker" output="screen">

    <!-- Camera parameters without distorsion -->
    <param name="square_size" value="0.12"/>        <!-- Square size in [m] corresponding to the distance between blobs center -->
    <param name="cam_px" value="1109.8389473004"/>  <!-- Intrinsic camera parameter px -->
    <param name="cam_py" value="1111.6863942763"/>  <!-- Intrinsic camera parameter py -->
    <param name="cam_u0" value="333.5848028848"/>   <!-- Intrinsic camera parameter u0 -->
    <param name="cam_v0" value="227.5683396488"/>   <!-- Intrinsic camera parameter v0 -->

    <!-- Target characteristics defined by 4 blobs in the corner -->
    <param name="square_size" value="0.125"/>       <!-- [meter] -->

    <!-- To connect image from v4l2_camera node-->
    <remap from="camera/image_raw" to="image_raw"/>
  </node>

</launch>
